modules/artifact-activation/src/main/java/org/mule/runtime/module/artifact/activation/api/plugin/PluginResolutionError.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.runtime.module.artifact.activation.api.plugin;
+
+/**
+ * Thrown to indicate that a given set of plugins was not resolved.
+ * 
+ * @since 4.5
+ */
+public class PluginResolutionError extends RuntimeException {
```

Comment:
where is this used?

Reply:
It's used in `BundlePluginDependenciesResolver`.